docs/library/array.rst

```diff
@@ -79,3 +79,8 @@ Classes
 
         **Note:** ``__repr__`` cannot be called directly (``a.__repr__()`` fails) and
         is not present in ``__dict__``, however ``str(a)`` and ``repr(a)`` both work.
+
+    Array objects also implement the buffer protocol, meaning the contents can
+    be accessed as raw bytes via a `memoryview` or other interfaces which use
+    this protocol.
```

Comment:
I reckon this is probably important enough that it could go up as the second paragraph of the `array.array` constructor, line 21.

Reply:
Good idea, have moved.